app/models/user.rb

```diff
@@ -49,6 +49,16 @@ class User < ActiveRecord::Base
   has_many :notes, -> { order 'created_at DESC' }
   has_many :deeds
 
+  has_many :random_collections,   -> { unrestricted.sample },
+    class_name: "Collection",  :foreign_key => "owner_user_id"
+  has_many :random_document_sets, -> { unrestricted.sample }, 
+    class_name: "DocumentSet", :foreign_key => "owner_user_id"
+
+  scope :owners,         -> { where(owner: true) }
+  scope :trial_owners,   -> { owners.where(account_type: 'Trial') }
+  scope :paid_owners,    -> { owners.where('paid_date > ?', Time.now) }
+  scope :expired_owners, -> { owners.where('paid_date <= ?', Time.now) }
+
```

Comment:
You could add these to the paid owners if you'd like.  I'd be OK with that.  (keeping out trial owners, but not others.)

Reply:
Changed these to `non_trial_owners.where(...)` so that we don't mistakenly suggest trial owners (or `nil` folks) are unpaid.